### ISSUE TITLE
Styling: Assess borders and box shadows in visualizations

### ISSUE THREAD

**Issue by jernestmyers** [VEuPathDB/web-eda#1270]

**Comment by asizemore** [comment]
@chowington and i had an interesting case in the subset download modal table where having a gray background created some accessibility issues (see [comment](https://github.com/VEuPathDB/web-eda/pull/1261#discussion_r927969442) and [other comment](https://github.com/VEuPathDB/web-eda/pull/1261#pullrequestreview-1051637865)). 

This made me realize that many parts of our tables when we have gray backgrounds do not pass accessibility rules. Even the variable coverage table has text that is technically inaccessible. To help alleviate these issues, i'd suggest we at least make the backgrounds of tables and plots white.

If we accept the above and make the backgrounds white, then we have (at least!) the options of no border, border, or drop shadow. I'm personally between the drop shadow and no border. The elevation suggests importance, but the birds eye and variable completeness table are of less importance than the main plot, which draws me towards the no border option. I also think putting both the birds eye and var coverage table within the same drop shadow box and giving that box a title ("Data completeness information"?) could be cool.

**Comment by asizemore** [comment]
i think over slack we decided to go with the drop shadows version. Related issue #[1286](https://github.com/VEuPathDB/web-eda/issues/1286)
